src/hooks/Projects.ts

```diff
@@ -125,6 +129,64 @@ export function useProjectsSearch(handle: string | undefined) {
   )
 }
 
+export function useTrendingProjects() {
+  const secondsInDay = 24 * 60 * 60
+
+  const payments = useSubgraphQuery(
+    {
+      first: 1000,
+      entity: 'payEvent',
+      keys: [
+        'amount',
+        {
+          entity: 'project',
+          keys: ['id'],
```

Comment:
Just to double check, we can't get more project data here?

Reply:
we could—it's just that this query doesn't need anything else.

it's matching the total volume of payments in the last 7 days to their respective projects, so for each payment it only needs the payment amount and its project id